README.md

```diff
@@ -25,8 +25,7 @@ The documentation is generated from the following resources
 - [CDK components, services, and directives](https://github.com/angular/components/tree/master/src/cdk)
 
 ## Development Setup
-1. Install Gulp globally: `npm i -g gulp`
-2. Run `npm install`
+1. Run `npm install`
```

Comment:
We're actually using `yarn` here now if you want to change it to that

Reply:
`README.md` refers to `npm` commands in several other places, so for the sake of consistency I would prefer to have `npm install`. If you would prefer to replace all usage of `npm` commands with their `yarn` equivalents I can close this PR and leave that task to someone else.